adhocteam/Head-Start-TTADP
# CD: Use DATABASE_URL

**Description of change**
Addresses feedback in https://github.com/HHS/Head-Start-TTADP/pull/234

I tried to follow what I think was the intention for the production sequelize setup in https://github.com/adhocteam/Head-Start-TTADP/blob/main/src/models/index.js#L12-L13
```
let sequelize;
if (config.use_env_variable) {
  sequelize = new Sequelize(process.env[config.use_env_variable], config);
} else {
  sequelize = new Sequelize(config.database, config.username, config.password, config);
}
```

**How to test**
- CI/CD pipeline works against sandbox, application logs are healthy and migrations still run as expected:
```
17:47:20.973: [APP/TASK/ec51485b.0] $ node_modules/.bin/sequelize db:migrate --options-path .production.sequelizerc
17:47:21.251: [APP/TASK/ec51485b.0] Sequelize CLI [Node: 12.20.0, CLI: 6.2.0, ORM: 5.22.3]
17:47:21.258: [APP/TASK/ec51485b.0] Loaded configuration file "build/config/config.js".
17:47:21.258: [APP/TASK/ec51485b.0] Using environment "production".
17:47:21.369: [APP/TASK/ec51485b.0] No migrations were executed, database schema was already up to date.
17:47:21.373: [APP/TASK/ec51485b.0] Done in 0.44s.
```

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/202

**Checklist**

- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
